frontend/sozisel-app/src/components/TemplatesList/TemplateList.tsx

```diff
@@ -24,11 +25,14 @@ function Alert(props: AlertProps) {
 
 export default function TemplateList(): ReactElement {
   const { t } = useTranslation("common");
+  const [successMsg, setSuccessMsg] = useState<string>("");
```

Comment:
Type `Message`.  Why did you name it like this but `includePublic` is not `includePub`?